ginga/misc/plugins/Cuts.py

```diff
@@ -514,10 +677,10 @@ def replot_all(self):
                 text.color = colors[0]
             #text.color = color
             self._replot(lines, colors)
-            if n:
-                self.w.delete_cut.set_enabled(True)
-                self.w.delete_all.set_enabled(True)
-                self.save_btn.set_enabled(True)
+
+        # Redraw slit image for selected cut
+        if self.cutstag != self._new_cut and not self.mdim_ref.is_playing():
+            self.redraw_slit('redraw')
```

Comment:
Here `mdim_ref` is not defined in the case I described above (open Cuts plugin after opening the file), so I get an exception raised and the Slit is not redrawn.